app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ApplicationsController < ApplicationController
+  def show
+    @application = Application.find(params[:id])
+    @pets = @application.pets
+    if params[:search] == "pet" && params[:pet_name].present? 
+      pet_name = params[:pet_name]
+      @search_pets = Pet.where("name ILIKE ?", "%#{pet_name}%")
```

Comment:
We don't want a search query with .where in the controller, as this would be an MVC violation. Controllers should only use the following CRUD activerecord methods: .create, .update, .destroy, .find, .find_by, .all

Also, this is a brownfield project so this method is actually already written for us! Check out the search method in the ApplicationRecord file. Alll models inherit from this parent class, so you can actually do `Pet.search(pet_name)` here and it would direct to that method